package/network/services/hostapd/files/hostapd.sh

```diff
@@ -874,6 +874,14 @@ wpa_supplicant_add_network() {
 		;;
 	esac
 
+	case "$wpa_cipher" in
+		GCMP)
+			append network_data "pairwise=GCMP" "$N$T"
+			append network_data "group=GCMP" "$N$T"
+		;;
+		*) ;;
```

Comment:
Had a similar case recently:
Is there a reason for defining an empty default case?